CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 All notable changes to the "azure-cosmosdb" extension will be documented in this file.
 
+## 0.13.0 - 2020-05-28
+### Added
+- PostgreSQL support
+  - Basic CRUD support added to PostgreSQL databases and integrated with the exisitng extension; updates to Tables is not yet supported
+  - Functions and stored procedures CRUD support
+### Changed
+- Top level naming convention is updated from "CosmosDB" to "Azure Databases"
```

Comment:
Don't forget to mention something like this:
```
- Viewing and querying Azure Cosmos DB graphs has been moved to a separate graph-specific extension
```

Reply:
Should we add a link to it?  It feels like the right thing to do, but I'm not sure we're exactly _trying_ to lead people to it?

Reply:
We handle this directly in-product when they click a graph. IMO that's the most important piece and I don't really have a preference for the docs/changelog

Reply:
I will just add it, just for record keeping purposes